add initial blog post about GCC (#279)
usegalaxy-eu/website: _posts/2019-08-01-gcc.md

---
site: freiburg
title: 'Galaxy Community Conference - a brief summary!'
tags: [training, tiaas, gcc]
supporters:
- galaxy-europe
- tiaas
- denbi
- elixir
---

The 2019 Galaxy Community Conference (GCC2019) was held in Freiburg, Germany, 1-8 July.
There was one day of training, three days of mixed meeting and training, and then two to four days of CollaborationFest.
The meeting featured joint & parallel sessions; invited, accepted, and lightning talks; poster and demo sessions;
birds-of-a-feather meetups; social events; and training sessions every day.

![GCC2019 Logo](/assets/media/gcc2019/gcc_logo_blue_bg.png)
![Group Picture](/assets/media/gcc2019/grouppicture.jpg)

With 231 participants, GCC2019 was the largest GCC ever with 39 Galaxy training courses, 65 talks, 54 posters,
13 demos and more than 30 different BoFs. Our participants came from 27 different countries of Eastern- and Western Europe,
North America, Asia, Oceania and Sub-saharan Africa.

![Group Picture](/assets/media/gcc2019/GCC2019_stats.png)

> GCC2019 has started on Monday, 1st July, with a dedicated training day for Galaxy beginners offering a user-centric track with “Introduction to using Galaxy” in parallel to the “Introduction to Galaxy Administration”. Participants interested in long read sequencing and some wet-lab work could visit and try the Nanopore loading zone where the Oxford Nanopore MinION was presented. In the evening, the first Birds-of-a-Feather (BoFs) were offered for special interest groups on Single Cell data analysis, Genome Annotation and Computational Chemistry.

Reviewer: ```suggestion
GCC2019 has started on Monday, 1st July, with a dedicated training day for Galaxy beginners offering a user-centric track with “Introduction to using Galaxy” in parallel to the admin and developer focused “Introduction to Galaxy Administration”. Participants interested in long read sequencing and some wet-lab work could visit and try the Nanopore loading zone where the Oxford Nanopore MinION was presented. In the evening, the first Birds-of-a-Feather (BoFs) were offered for special interest groups on Single Cell data analysis, Genome Annotation, and Computational Chemistry.
```

Author: Sorry, can not take this over, merged it to late :(

![Schedule Monday](/assets/media/gcc2019/sched_mon.png)

> GCC2019 was officially opened on Tuesday, 2nd July, by Björn Grüning and Anika Erxleben. One of our Galaxy power users, Claudia Isabelle Keller Valsecchi, was invited to present her impressive data on “Evolution of X chromosome recognition during Drosophila Dosage Compensation. After some talks on Galaxy Australia, Machine learning, and Galaxy Workflows, talks and workshops were offered in two tracks, one more user-driven, the other more admin-related. Galaxy is used in various fields of research, as we heard e.g. from Anne Fouilloux talking about Climate analysis with Galaxy. During the afternoon break, the participants could discover interesting Galaxy projects in the poster and demo session. The Epigenetics sessions showed how to perform 16S analysis with mothur, ChIPseq data analysis and HiC data analysis. Beside multi-omics data analysis in Galaxy, the first time Machine learning with Galaxy was presented.

Reviewer: ```suggestion
GCC2019 was officially opened on Tuesday, 2nd July, by Björn Grüning and Anika Erxleben. One of our Galaxy power users, Claudia Isabelle Keller Valsecchi, was invited to present her impressive data on “Evolution of X chromosome recognition during Drosophila Dosage Compensation. After some talks on Galaxy Australia, Machine learning, and Galaxy Workflows, talks and workshops were offered in two tracks, one more user-driven, the other more admin-related. Galaxy is used in various fields of research, as we heard e.g. from Anne Fouilloux talking about Climate analysis with Galaxy. During the afternoon break, the participants could discover interesting Galaxy projects in the poster and demo session. The Epigenetics sessions showed how to perform 16S analysis with mothur, ChIPseq data analysis and HiC data analysis. In addition to multi-omics data analysis in Galaxy, Machine learning with Galaxy was presented for the first time.
```

![Schedule Wednesday](/assets/media/gcc2019/sched_wed.png)

Proteomics, metabolomics, metagenomics, and variant calling data analysis were presented on Thursday in various talks, workshops, demos and posters, such as microbiome data, ProteoRE, IRIDA, and Workflow4Metabolomics. With Galaxy-E Yvan brought us our “Wonderful World” a bit closer. He presented the Galaxy Ecology project with newly established dedicated Galaxy subdomain. Before closing the conference, Bérénice gave an update on the Galaxy training network. 
The GCC2019 core conference days were followed by 4 days of CollaborationFest. During these 4 days around 100 people were working on e.g. library enhancements, Galaxy packages,
development of an ATAC-seq tutorial, create backend and frontend for select parameters in workflows, tool menu style, EDAM in Galaxy, Training development kit, and on a lot of work on usegalaxy.*.

![Schedule Thursday](/assets/media/gcc2019/sched_thu.png)

## Freiburg Challenge

![Group Picture](/assets/media/gcc2019/freiburg_challenge.png)

## Thank you: Scientific Program Committee

> Matthias Bernt, Helmholtz Centre for Environmental Research – UFZ

Reviewer: Maybe a bulleted list here, they all run together on one line.

Author: Done.

---
site: freiburg
title: 'Galaxy Community Conference - a brief summary!'
tags: [training, tiaas, gcc]
supporters:
- galaxy-europe
- tiaas
- denbi
- elixir
- aws
---

The [2019 Galaxy Community Conference (GCC2019)](https://galaxyproject.org/events/gcc2019) was held in Freiburg, Germany, 1-8 July. 2019 was the largest GCC ever with 231 participants, 39 Galaxy training courses, 65 talks, 54 posters,
13 demos and more than 30 different BoFs. Our participants came from 27 different countries of Eastern- and Western Europe,
North America, Asia, Oceania and Sub-saharan Africa.

> ![GCC2019 Logo](/assets/media/gcc2019/gcc_logo_blue_bg.png)

Author: We could also remove the logo, does not fit so nicely here.

---
site: freiburg
title: 'Galaxy Community Conference - a brief summary!'
tags: [training, tiaas, gcc]
supporters:
- galaxy-europe
- tiaas
- denbi
- elixir
- aws
---

The [2019 Galaxy Community Conference (GCC2019)](https://galaxyproject.org/events/gcc2019) was held in Freiburg, Germany, 1-8 July. 2019 was the largest GCC ever with 231 participants, 39 Galaxy training courses, 65 talks, 54 posters,

> 13 demos and more than 30 different BoFs. Our participants came from 27 different countries of Eastern- and Western Europe,

Reviewer: ```suggestion
13 demos, and more than 30 different BoFs. Our participants came from 27 different countries of Eastern- and Western Europe,
```

* Expanded from two days to three days.
* Offered dual presentation tracks most of the time
* Intermixed training and presentation sessions.

These changes supported the record amount of content that was offered this year and they made the meeting more interactive than in past years.
The meeting featured plenary & parallel sessions; invited, accepted, and lightning talks; [poster](https://gcc2019.sched.com/overview/type/C.+Posters+%2F+Demos+%2F+Sponsors/Posters) and [demo](https://gcc2019.sched.com/overview/type/C.+Posters+%2F+Demos+%2F+Sponsors/Demos) sessions;
[birds-of-a-feather](https://gcc2019.sched.com/overview/type/D.+Break+%2F+Social+%2F+Networking+%2F+Meal/BoFs) meetups; [social events](https://gcc2019.sched.com/overview/type/D.+Break+%2F+Social+%2F+Networking+%2F+Meal); and training sessions every day.

![Stats](/assets/media/gcc2019/GCC2019_stats.png)

Have a look at our [conference book](https://github.com/usegalaxy-eu/gcc2019/raw/master/com/program/book.pdf) to get all information.

> GCC2019 started on [Monday](https://gcc2019.sched.com/2019-07-01/overview), July 1st, with a dedicated training day for Galaxy beginners offering a user-centric track with [“Introduction to using Galaxy”](https://gcc2019.sched.com/event/Lud9/introduction-to-using-galaxy-galaxy-101) in parallel to the admin and developer focused [“Introduction to Galaxy Administration”](https://gcc2019.sched.com/event/LudC/introduction-to-galaxy-administration-i). Participants interested in long read sequencing and some wet-lab work could visit and try the [Nanopore loading zone](https://gcc2019.sched.com/event/QZzb/nanopore-loading-zone) with the Oxford Nanopore MinION. In the evening, the first Birds-of-a-Feather (BoFs) were offered for attendees interested in Single Cell data analysis, Genome Annotation, and Computational Chemistry.

Reviewer: ```suggestion
GCC2019 started on [Monday](https://gcc2019.sched.com/2019-07-01/overview), July 1st, with a dedicated training day for Galaxy beginners offering a user-centric track with [“Introduction to using Galaxy”](https://gcc2019.sched.com/event/Lud9/introduction-to-using-galaxy-galaxy-101) in parallel to the admin and developer focused [“Introduction to Galaxy Administration”](https://gcc2019.sched.com/event/LudC/introduction-to-galaxy-administration-i). Participants interested in long read sequencing and wet-lab work could visit [Nanopore loading zone](https://gcc2019.sched.com/event/QZzb/nanopore-loading-zone) and try the Oxford Nanopore MinION. In the evening, the first Birds-of-a-Feather (BoFs) were offered for attendees interested in Single Cell data analysis, Genome Annotation, and Computational Chemistry.
```

rewrote the bit around visit nanopore, removed 'some' from wet-lab work

![Schedule Tuesday](/assets/media/gcc2019/sched_tue.png)

> [Wednesday\'s](https://gcc2019.sched.com/2019-07-03/overview) Galaxy user track was dedicated to transcriptomics and genomics data analysis. The demand for single cell RNA-seq data analysis was very obvious in the presentations by [Mehmet](https://gcc2019.sched.com/event/PSeu/single-cell-rna-analysis-in-galaxy-on-10x-datasets), [Christophe](https://gcc2019.sched.com/event/PSex/biology-driven-classification-of-single-cells-rnaseq-data-in-galaxy), and [Jonathan](https://gcc2019.sched.com/event/PSfF/galaxy-container-for-hinxton-single-cell-interactive-analysis-portal-hisciap). Running Galaxy for annotation with e.g. [Apollo](https://gcc2019.sched.com/event/PSfO/visual-refinement-of-genome-annotations-with-apollo-in-a-community-environment), [G-ONRamp](https://gcc2019.sched.com/event/PSfR/architectural-and-systems-improvements-to-g-onramp-a-collaborative-eukaryotic-genome-annotation-platform), and  [ChiRA](https://gcc2019.sched.com/event/PSfU/chira-an-integrated-framework-for-chimeric-read-annotation) was successfully shown in talks, posters, trainings and demos. Galaxy also now runs on [Kubernetes](https://gcc2019.sched.com/event/LufN/running-galaxy-on-kubernetes). Before the conference dinner, a public science project was presented by the [Street Science Community](https://gcc2019.sched.com/event/PSfv/street-science-community-make-science-tangible-accessible-by-bringing-it-to-the-street-with-open-science-workshops). More than 150 people joined us for the conference dinner in [Augustiner im Bankepeter](http://www.augustiner-freiburg.de). We spent an amazing evening in the beer garden with our friends and colleagues. Thank you all!

Reviewer: ```suggestion
[Wednesday\'s](https://gcc2019.sched.com/2019-07-03/overview) Galaxy user track was dedicated to transcriptomics and genomics data analysis. The demand for single cell RNA-seq data analysis was very obvious in the presentations by [Mehmet](https://gcc2019.sched.com/event/PSeu/single-cell-rna-analysis-in-galaxy-on-10x-datasets), [Christophe](https://gcc2019.sched.com/event/PSex/biology-driven-classification-of-single-cells-rnaseq-data-in-galaxy), and [Jonathan](https://gcc2019.sched.com/event/PSfF/galaxy-container-for-hinxton-single-cell-interactive-analysis-portal-hisciap). Running Galaxy for annotation with e.g. [Apollo](https://gcc2019.sched.com/event/PSfO/visual-refinement-of-genome-annotations-with-apollo-in-a-community-environment), [G-ONRamp](https://gcc2019.sched.com/event/PSfR/architectural-and-systems-improvements-to-g-onramp-a-collaborative-eukaryotic-genome-annotation-platform), and [ChiRA](https://gcc2019.sched.com/event/PSfU/chira-an-integrated-framework-for-chimeric-read-annotation) was successfully shown in talks, posters, trainings, and demos. Galaxy also now runs on [Kubernetes](https://gcc2019.sched.com/event/LufN/running-galaxy-on-kubernetes). Before the conference dinner, a public science project was presented by the [Street Science Community](https://gcc2019.sched.com/event/PSfv/street-science-community-make-science-tangible-accessible-by-bringing-it-to-the-street-with-open-science-workshops). More than 150 people joined us for the conference dinner in [Augustiner im Bankepeter](http://www.augustiner-freiburg.de). We spent an amazing evening in the beer garden with our friends and colleagues. Thank you all!
```

oxford comma
